docs/Jenkinsfile

```diff
@@ -142,6 +142,22 @@ podTemplate(label: label) {
 
                             sh "docker tag $authorization_and_authentication-$docs_image_suffix ${dockerPushRoot}$authorization_and_authentication-$docs_image_suffix:$dockerImageTag"
                             sh "docker push ${dockerPushRoot}$authorization_and_authentication-$docs_image_suffix:$dockerImageTag"
+                            if (isMaster) {
+                                sh "docker push ${dockerPushRoot}$kyma-$docs_image_suffix:latest"
```

Comment:
These images should be built or tagged first with `latest` before pushing.

Reply:
Yes definitely, missed that! Fixed now thanks!